### DISCUSSION TITLE
Serviços

### DISCUSSION THREAD

**Discussion by Fesantt** [RamonSilva20/mapos#2074] [Q&A]
![image](https://user-images.githubusercontent.com/33626716/223123512-b2937e0c-5c30-433a-b6b8-446432a7a6e0.png)
queria saber se é possivel adicionar serviços a ordem sem que os mesmos estejam previamente cadastrados, caso n seja possivel alguem poderia me ajudar a fazer essa alteração

**Comment by Stephanienogueira** [comment]
Olá Felipe, estou nessa também.
Eu consegui alterar o código para inserir serviços com o valor zerado, porém isso impacta no valor total.

Vou te informar o que eu fiz caso você queira fazer também.

Para desativar a validação obrigatória de preço no serviço.
1 passo: abra o arquivo "editarservico.php localizado na pasta "views/serviços/

2 passo: localize a linha que contém a regra de validação para o campo preço:

rules:{
    nome:{
        required:true
},
Preco:{deixa em branco

}

OBSERVAÇÃO:
Isso impacta no valor total da ordem de serviço oi orçamento

**Comment by Fesantt** [comment]
Não teria alguma forma sem impactar no valor total da ordem de serviço, tipo eu pudesse cadastrar o valor ali na hora

**Comment by diegoandrade93** [comment]
mais facil colocar um modal ali ?
